packages/scene-graph-mediator/runtime/src/interface/ImporterPlugin.ts

```diff
@@ -2,6 +2,7 @@ import { SchemaJson, Node } from '@drecom/scene-graph-schema';
 import { ImportOption } from 'importer/Importer';
 
 export default interface ImporterPlugin {
+  createRuntimeObject?(node: Node, resources: any): any | null;
```

Comment:
Please clarify the behavior with multiple plugin that implements `createRuntimeObject`.

Reply:
added comment about when multiple plugin set.
2f6ced4ce0c80bc8e8de8de0fec02d5a5186a718